app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -64,7 +65,8 @@ export class PublishRepository extends React.Component<
 
   private async fetchOrgs(account: Account) {
     const api = API.fromAccount(account)
-    const orgs = await api.fetchOrgs()
+    const orgs = [...(await api.fetchOrgs())]
```

Comment:
On second thought, I think we should go for a more TypeScripty way of doing this.

```ts
const orgs = await api.fetchOrgs() as IAPIUser[]

orgs.sort((a, b) => compare(a.login, b.login))
```